src/sentry/api/event_search.py

```diff
@@ -766,32 +769,35 @@ def get_filter(query=None, params=None):
         "group_ids": [],
     }
 
-    def get_projects(params):
-        return {
-            p["slug"]: p["id"]
-            for p in Project.objects.filter(id__in=params.get("project_id", [])).values(
-                "id", "slug"
-            )
-        }
-
     def to_list(value):
         if isinstance(value, list):
             return value
         return [value]
 
-    projects = None
     for term in parsed_terms:
         if isinstance(term, SearchFilter):
             name = term.key.name
-            if name == PROJECT_KEY:
-                if projects is None:
-                    projects = get_projects(params)
-                condition = ["project_id", "=", projects.get(term.value.value)]
-                kwargs["conditions"].append(condition)
-            elif name == "issue.id" and term.value.value != "":
+            if name in (PROJECT_ALIAS, PROJECT_NAME_ALIAS):
```

Comment:
After our recent trouble with trace, I was thinking we should update the url generation for tag pills on issue details to apply the `tags[]` wrapper so that we don't cause conflicts with people's `project` tags.